remoulade/helpers/actor_arguments.py

```diff
@@ -0,0 +1,24 @@
+from inspect import Parameter, signature
+
+
+def get_actor_arguments(actor):
+    params = signature(actor.fn).parameters
+
+    def parsetype(rawtype):
+        if str(rawtype).startswith("typing"):
+            return str(rawtype)
+        try:
+            return str(rawtype.__name__)
+        except AttributeError:
+            return str(rawtype)
+
+    args = []
+    for param in params.values():
+        arg = {"name": param.name}
+        if param.annotation != Parameter.empty:
+            arg["type"] = parsetype(param.annotation)
+        if param.default != Parameter.empty:
+            arg["default"] = str(param.default)
```

Comment:
does it work for `None` ?

Reply:
I added a test to make sure it does. The only problem is that there is no way to differentiate between None and "None" as it is parsed as JSON.

Reply:
I was thinking about replacing it by an empty string, as this is what you usually want (but we will have to see how to manage None values anyway)